components/operator/internal/state/registry_test.go

```diff
@@ -345,4 +346,53 @@ func Test_addRegistryConfigurationWarnings(t *testing.T) {
 		addRegistryConfigurationWarnings(&extRegSecret, []corev1.Secret{}, s)
 		require.Equal(t, "", s.warningBuilder.Build())
 	})
+
+	t.Run("internal registry pvc size", func(t *testing.T) {
```

Comment:
Do we have tests for other cases - especially negative ones (e.g. `PV` with an empty `Size` or changing `Size` to a lower value)?